examples/logconcave/generalized_hyperbolic.r

```diff
@@ -0,0 +1,65 @@
+# VolEsti (volume computation and sampling library)
```

Comment:
Why `R-proj/man` and not some other directory? It seems that this directory contains only auto-generated filed by roxygen with extension `.Rd`.

Reply:
Maybe we could create a new folder in `R-proj/inst/examples`. `R-proj/man` directory is about small auto-generated examples written before function declaration; It would also increase the run-time of CRAN tests/examples for which there is a time limt.

Reply:
@vissarion @TolisChal  Currently I placed the R examples to `R-proj/examples`. I believe that this is the most correct way to present them since the `examples/` directory also appears at the cpp repo, so if you want one-day to separate the cpp project from the R project that should mean that they must have the same high-level structure.
See https://github.com/stan-dev/pystan and https://github.com/stan-dev/stan.